### DISCUSSION TITLE
wire:initial-data="{"

### DISCUSSION THREAD

**Discussion by elfeffe** [livewire/livewire#3304] [7. Bugs (archived)]
I'm not sure if it's a bug, but I can't see how can I debug why I'm having that.
In any case, it fires an error when it tries to parse JSON.
I'm using 2.5.3

**Comment by mixdd** [comment]
Looks similar to the issue I'm having with setting an attribute value to JSON after 2.5.2 release. /pull/3152
Issue does not happen for me in 2.5.1, so perhaps downgrading would fix for you as well.

**Reply by elfeffe** [reply]
The problem occurs when I try to do a Livewire.restart()
If I use 2.5.1 I get this instead:
`** TypeError: Cannot read property 'fingerprint' of null**`

And with 2.5.3 I get that empty JSON that causes:
`Uncaught SyntaxError: Unexpected end of JSON input`

I'm loading a component inside another one, and after that, I want to move the div to take it out of the original component, but LW stops working on the child one, and then I want to restart to see if that helps.

**Comment by joshhanley** [comment]
@elfeffe can you share a screenshot of the error in the console and your component that's causing that?

I've been track a few issues like that, so just want to confirm it's the same

Thanks!

**Reply by elfeffe** [reply]
I have updated to the latest version, initial-data is filled now, but I have a new error (I had the same error with version 2.5.1):
```
livewire.js?id=83b555bb3e243bc25f35:13 Uncaught TypeError: Cannot read property 'fingerprint' of null
    at new Component (livewire.js?id=83b555bb3e243bc25f35:13)
    at livewire.js?id=83b555bb3e243bc25f35:13
    at livewire.js?id=83b555bb3e243bc25f35:13
    at walk (livewire.js?id=83b555bb3e243bc25f35:1)
    at walk (livewire.js?id=83b555bb3e243bc25f35:1)
    at walk (livewire.js?id=83b555bb3e243bc25f35:1)
    at walk (livewire.js?id=83b555bb3e243bc25f35:1)
    at walk (livewire.js?id=83b555bb3e243bc25f35:1)
    at walk (livewire.js?id=83b555bb3e243bc25f35:1)
    at walk (livewire.js?id=83b555bb3e243bc25f35:1)
Component @ livewire.js?id=83b555bb3e243bc25f35:13
(anonymous) @ livewire.js?id=83b555bb3e243bc25f35:13
(anonymous) @ livewire.js?id=83b555bb3e243bc25f35:13
walk @ livewire.js?id=83b555bb3e243bc25f35:1
walk @ livewire.js?id=83b555bb3e243bc25f35:1
walk @ livewire.js?id=83b555bb3e243bc25f35:1
walk @ livewire.js?id=83b555bb3e243bc25f35:1
walk @ livewire.js?id=83b555bb3e243bc25f35:1
walk @ livewire.js?id=83b555bb3e243bc25f35:1
walk @ livewire.js?id=83b555bb3e243bc25f35:1
value @ livewire.js?id=83b555bb3e243bc25f35:13
value @ livewire.js?id=83b555bb3e243bc25f35:13
Component @ livewire.js?id=83b555bb3e243bc25f35:13
(anonymous) @ livewire.js?id=83b555bb3e243bc25f35:13
value @ livewire.js?id=83b555bb3e243bc25f35:13
value @ livewire.js?id=83b555bb3e243bc25f35:13
(anonymous) @ VM103540:1
```

It happens when I try to run Livewire.restart()
It works when I load the page, it seems that it finds the initial-data, but if I try to run that after page load, initial-data doesn't exist (it was "{" on the previous version), and then it fails.

**Comment by joshhanley** [comment]
@elfeffe and @mixdd a fix has been tagged for this, please upgrade to Livewire 2.5.5 and let us know if you are still having any issues 🙂

**Reply by elfeffe** [reply]
My new error, is related to the latest 2.5.5 
`TypeError: Cannot read property 'fingerprint' of null`

**Reply by joshhanley** [reply]
@elfeffe can you share your component class and blade view?
